spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/export/otlp/OtlpPropertiesConfigAdapterTests.java

```diff
@@ -67,4 +68,17 @@ void whenPropertiesHeadersIsSetAdapterHeadersReturnsIt() {
 		assertThat(new OtlpPropertiesConfigAdapter(properties).headers()).containsEntry("header", "value");
 	}
 
+	@Test
+	void whenPropertiesBaseTimeUnitIsNotSetAdapterBaseTimeUnitReturnsMillis() {
+		OtlpProperties properties = new OtlpProperties();
+		assertThat(new OtlpPropertiesConfigAdapter(properties).baseTimeUnit()).isSameAs(TimeUnit.MILLISECONDS);
+	}
```

Comment:
wonder if this assertions should be moved to https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/export/otlp/OtlpPropertiesTests.java

Reply:
That's a good point, I think this is valid in itself but there is no default in `OtlpProperties` so I set it and added an assertion to OtlpPropertiesTests.
Similarly to aggregation temporality.